src/connectionmanager.js

```diff
@@ -936,27 +756,8 @@ export default class ConnectionManager {
 
             const credentials = credentialProvider.credentials();
             options = options || {};
-            if (credentials.ConnectAccessToken && options.enableAutoLogin !== false) {
-
-                ensureConnectUser(credentials).then(() => {
-
-                    if (server.ExchangeToken) {
-                        addAuthenticationInfoFromConnect(server, serverUrl, credentials).then(() => {
-
-                            afterConnectValidated(server, credentials, systemInfo, connectionMode, serverUrl, true, options, resolve);
-
-                        }, () => {
-
-                            afterConnectValidated(server, credentials, systemInfo, connectionMode, serverUrl, true, options, resolve);
-                        });
-
-                    } else {
+            if (options.enableAutoLogin !== false) {
```

Comment:
why this `if` is needed? that `else` clause before was executed even when that condition was false

Reply:
https://github.com/jellyfin/jellyfin-apiclient-javascript/pull/18/commits/e808983d23bba0c5f56739c0c0d63b5d387ee3bd